Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Audio/GetAllAudioTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.MediatR.Media.Audio.GetAll;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System.Linq.Expressions;
+using Xunit;
+
+using Entities = Streetcode.DAL.Entities.Media;
+
+namespace Streetcode.XUnitTest.MediatRTests.Media.Audio
+{
+    public class GetAllAudioTests
+    {
+        [Fact]
+        public async Task GetAllAudio_Success()
+        {
+            var repository = new Mock<IRepositoryWrapper>();
+            var mockMapper = new Mock<IMapper>();
+
+            var testAudioList = new List<Entities.Audio>()
+            {
+                new Entities.Audio() {Id = 1 }
+            };
+            var testAudioListDTO = new List<AudioDTO>()
+            {
+                new AudioDTO() {Id = 1 }
+            };
+            var testAudio = new Entities.Audio() { Id = 1 };
+
+            repository.Setup(repo => repo.AudioRepository
+                .GetFirstOrDefaultAsync(It.IsAny<Expression<Func<Entities.Audio, bool>>>(), It.IsAny<Func<IQueryable<Entities.Audio>, IIncludableQueryable<Entities.Audio, Entities.Audio>>?>()))
+                .ReturnsAsync(testAudio);
+
+            repository.Setup(repo => repo.AudioRepository
+                .GetAllAsync(null, null)).ReturnsAsync(testAudioList);
```

Comment:
Can we use here It.IsAny, instead of nulls?

Reply:
Yes, we can use It.IsAny, replaced